src/current/cockroachcloud/serverless-cluster-management.md

```diff
@@ -49,11 +49,11 @@ The monthly cost estimate is calculated using simple extrapolation that assumes
 1. In the **Usage this month** section of your cluster's [**Overview** page](cluster-overview-page.html), click **Estimate usage cost**.
 1. Select a time frame in which your workload was active.
 
-    Your used [RUs](plan-your-cluster-serverless.html#request-units), used storage, and spend from the selected time frame will be shown along with your monthly cost estimate. The spend and cost estimate do not account for the [free resources](plan-your-cluster-serverless.html#free-vs-paid-usage) granted to every organization, which you would have to use up before being charged.
+    Your used [RUs](plan-your-cluster-serverless.html#request-units), used storage, and spend from the selected time frame will be shown along with your monthly cost estimate. The spend and cost estimate do not account for the [free resources](plan-your-cluster-serverless.html#free-vs-paid-usage) granted to each non-contract organization, which you would have to use up before being charged.
```

Comment:
```suggestion
    Your used [RUs](plan-your-cluster-serverless.html#request-units), used storage, and spend from the selected time frame will be shown along with your monthly cost estimate. The spend and cost estimate do not account for the [free resources](plan-your-cluster-serverless.html#free-vs-paid-usage) granted to each non-contract organization, which you would have to use up before being charged.
```
Can you use a different noun than "spend", like "costs"? How about "The number of RUs used for queries and storage displays, along with your costs for the period."